### PR TITLE
add callout to split-jobs lesson

### PR DESCRIPTION
**Pull request by andriiusachov**
Added callout about ignoremissing and maxfiles arguments of the splitter

### CODE REVIEW

File: first-analysis-steps/split-jobs.md [hunk 1]
```diff
@@ -28,6 +28,12 @@ and reduces the impact of jobs failing due to grid problems.
 Setting `filesPerJob=5` should work well for real data, while `filesPerJob=1` should be good for signal MC.
 {% endcallout %} 
 
+{% callout Splitter arguments %}
```
Comment (vlisovsk) [inline first-analysis-steps/split-jobs.md]: I think you are missing the "quotation marks" around the "Splitter arguments". Probably that's why CI failed

Comment (andriiusachov) [reply first-analysis-steps/split-jobs.md]: indeed, thanks

File: first-analysis-steps/split-jobs.md [hunk 2]
```diff
@@ -28,6 +28,12 @@ and reduces the impact of jobs failing due to grid problems.
 Setting `filesPerJob=5` should work well for real data, while `filesPerJob=1` should be good for signal MC.
 {% endcallout %} 
 
+{% callout "Splitter arguments" %}
+The splitter has other useful arguments: 
+- `maxFiles`      : the maximal total number of files. By default splitter will run over all files in the dataset (corresponds to default value -1)
```
Comment (alexpearce) [inline first-analysis-steps/split-jobs.md]: * "by default _the_ splitter"
* "_which_ corresponds to _the_ default value _of_ -1"

File: first-analysis-steps/split-jobs.md [hunk 3]
```diff
@@ -28,6 +28,12 @@ and reduces the impact of jobs failing due to grid problems.
...
+- `ignoremissing` : boolean indicating whether it is appropriate to run if there are data files, which are not accessible at the moment. The later is important if it is necessary to make sure that resulting tuples correspond to whole data/MC sample.
```
Comment (alexpearce) [inline first-analysis-steps/split-jobs.md]: * I would remove the comma here "are data files which are".
* "The later" -> "This is"
* "that _the_ resulting _ntuples_ correspond to _the_ whole"

Comment (andriiusachov) [reply first-analysis-steps/split-jobs.md]: Thanks! Done

**Comment by chrisburr** [comment]
I think the comments were all dealt with so LGTM, thanks! 👍

**Pull request merged**